bitshares_pricefeed/sources/coincap.py

```diff
@@ -18,10 +18,8 @@ def _fetch(self):
                 coincap_global = requests.get('http://www.coincap.io/global').json()
                 alt_cap = float(coincap_global["altCap"])
                 alt_caps_x = [float(coin['mktcap'])
-                              for coin in coincap_front
-                              if 'position24' in coin and
-                              int(coin['position24']) <= 11 and
-                              coin['short'] != "BTC"]
+                              for coin in coincap_front[0:11]
```

Comment:
Can we make that `11` and the `10` a parameter maybe?

Reply:
Yep, it should be a named constant. I'll change it.

Reply:
see 
e5f2551